.cloud-build/execute_changed_notebooks_cli.py

```diff
@@ -129,26 +140,67 @@ def str2bool(v):
     base_branch=args.base_branch,
 )
 
+
+def _load_results() -> List[Dict[str, Any]]:
```

Comment:
The cli is just a wrapper around `execute_changed_notebooks_helper.get_changed_notebooks`. Move any business logic into `execute_changed_notebooks_helper`

Reply:
moved